### ISSUE TITLE
series_name should be updated with English-equivalent, if it exists, by tvdb_lookup

### ISSUE THREAD

**Issue by mfonville** [Flexget/Flexget#1459]
### Expected behaviour:
A series has a localized name, using the localized name it can be found in `tvdb_lookup`, other plugins expect the English name for further processing.
`tvdb_lookup` is in the perfect position to replace series_name with its English variant, if it exists, so that e.g. `trakt_lookup` can find the series too.

It might be an idea to keep the original localized name around too, or e.g. load all tvdb localized names in an array.

### Actual behaviour:
`series_name` is not being replaced at all, other plugins can't process non-English information

### Additional information:

- Flexget Version: 2.4.2

**Comment by liiight** [comment]
thetvdb does not return english name from its API, only the localized name:
```json
{
  "data": [
    {
      "aliases": [
        "VPRO Tegenlicht"
      ],
      "banner": "graphical/252712-g2.jpg",
      "firstAired": "2002-09-08",
      "id": 252712,
      "network": "VPRO",
      "overview": "Tegenlicht is de reeks informatieve programma's van VPRO Televisie, waarin nieuwe ideeën en trends worden onderzocht binnen de wereld van politiek, economie, maatschappij en wetenschap. \r\nTegenlicht beschouwt zichzelf nadrukkelijk als de eerste en enige future affairs-rubriek binnen het bestel. Het programma nestelt zich daarmee aan de frontlinie, en zoekt zijn verhalen op die plekken waar ideeën worden ontwikkeld, getest en bekritiseerd. Via niet voor de hand liggende, controversiële en tegelijkertijd gedegen analyses wil Tegenlicht zijn kijk geven op de wereld; zowel op nationale als op internationale ontwikkelingen die onze wereld in de 21ste eeuw vormgeven.\r\n\r\nTegenlicht is een programma zonder een vast 'format'. Afhankelijk van het onderwerp wordt steeds een passende vorm gekozen, waardoor langere reportages uit binnen- en buitenland worden afgewisseld met debatten, en met uitzendingen waarbij één persoon alle ruimte krijgt zijn of haar visie op een onderwerp te geven.",
      "seriesName": "Tegenlicht",
      "status": "Continuing"
    }
  ]
}
```
so that's not possible without doing another lookup based on the id, which is starting be awkward

**Comment by mfonville** [comment]
aha, that is indeed troublesome. And indeed not something that should a default, maybe it is worth to make it an extra optional feature (or even separate plugin) to lookup a series again in another language.

I guess more useful would be to e.g. make in that case the `trakt_lookup` not prefer to search by name, but to let it prefer to search by available ids like `tvdb_id` or `imdb_id` during lookup. Those are more reliable than various localized names.

**Comment by liiight** [comment]
I told you in the other issue, it already lookup via ID if present. Proof:
```yaml
tasks:
  tvdb_test:
    mock:
      - {title: 'bdsfdfgsdf', tvdb_id: 252712}
    trakt_lookup: yes
    accept_all: yes
    exec: "echo '{{ trakt_series_name}}'"
```
```
2016-10-21 14:47 VERBOSE  details       tvdb_test       Produced 1 entries.
2016-10-21 14:47 VERBOSE  task          tvdb_test       ACCEPTED: `bdsfdfgsdf` by accept_all plugin
2016-10-21 14:47 VERBOSE  details       tvdb_test       Summary - Accepted: 1 (Rejected: 0 Undecided: 0 Failed: 0)
2016-10-21 14:47 VERBOSE  exec          tvdb_test       Executing: echo 'Backlight'
2016-10-21 14:47 INFO     exec          tvdb_test       Stdout: b"'Backlight'\r\n"
```

**Comment by mfonville** [comment]
ok, my fault for not understanding it already did, sorry for that.
I will just use the `plugin_priority` to solve this quirk then!

**Comment by mfonville** [comment]
I am not sure how/why it happens, and I will investigate more, but the lookup of 'Zondag met Lubach' always fails via `tvdb_lookup` fails.
I have set the `plugin_priority` to first check thetvdb before trakt.
But still, for this specific show it results in no `tvdb_id` set, which then results in failing the `trakt_lookup`.
I now added
```
if:
      - "series_name == 'Zondag met Lubach'":
          set:
            tvdb_id: 288799
```
to the config, that manually sets the `tvdb_id` and the lookup works fine. :-/ I am flabbergasted
